scripts/network.js

```diff
@@ -116,28 +118,29 @@ if (networkEnabled) {
     request.send();
   }
 
-var sendTransaction = function(hex, msg = '') {
-    const request = new XMLHttpRequest();
-    request.open('POST', cExplorer.url + "/api/v2/sendtx/", true);
-    request.onerror = networkError;
-    request.onreadystatechange = function () {
+ var sendTransaction = async function(hex, msg = '') {
+    return new Promise(function (resolve, reject) {
```

Comment:
We can use fetch instead of making a new promise

Reply:
fixed